rails_event_store/spec/protobuf_integration_spec.rb

```diff
@@ -4,8 +4,10 @@
 class AsyncProtoHandler < ActiveJob::Base
   self.queue_adapter = :inline
 
-  def perform(event)
-    @@event = YAML.load(event)
+  cattr_accessor :event_store
```

Comment:
not ideal but I don't have better idea right now.

I prefer our tests to use similar constructs as devs would use for their application: `Rails.configuration.event_store` but that's not easy to achieve here.

Reply:
Some global scope is necessary here, maybe just `EventStore` as it is kind of similar to `Rails`